amazon-sqs/5.8/modules/ROOT/pages/amazon-sqs-connector-reference.adoc

```diff
@@ -51,31 +50,31 @@ Anypoint Connector for Amazon SQS (Amazon SQS Connector) enables you to interfac
 | Workstation a| String |  The optional proxy workstation. |  | 
 |===
 [[config_role]]
-===== Role
+==== Role
 
 
 ====== Parameters
 [%header,cols="20s,20a,35a,20a,5a"]
 |===
 | Name | Type | Description | Default Value | Required
-| Role ARN a| String |  The Role ARN unique identifies role to assume to gain cross account access. |  | x
-| Test Queue ARN a| String |  This parameter is used to test connection to SQS connector. If you do not specify this parameter the test connection is not performed. A queue ARN is constructed in the following format: arn:aws:{REGION}:{ACCOUNT_NUMBER}:{QUEUE_NAME}.  Example: arn:aws:sqs:us-east-1:123456789101:test-queue |  | 
+| Role ARN a| String | Role ARN that uniquely identifies the role to assume to gain cross-account access |  | x
```

Comment:
Period because of full sentences in same table column
```suggestion
| Role ARN a| String | Role ARN that uniquely identifies the role to assume to gain cross-account access. |  | x
```